README.md

```diff
@@ -7,6 +7,15 @@ Our BE services all have fundamentally the same github actions workflows copy &
 
 This repository introduces DRY Github Action workflows that can be used across repositories.
 
+## Tags
+From time to time there will be some breaking changes introduced to `properly-workflows`. To make the task of upgrading/ downgrading easier this repo makes use of git tags. This repo uses [annotated tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags), which typically include a message indicating why the tag change occurred.
+
+It is recommended to create a minor tag when adding any new functionality, but is required to add a major tag when a breaking change is added, as per [semantic versioning](https://semver.org/).
+To add a tag on branch (e.g. after changes have been merged to `main`):
+```
+git tag -a v1.0 -m "added a breaking change that does X and Y"
```

Comment:
Should we alternatively suggest using GitHub to add them like so https://github.com/GoProperly/properly-workflows/tags

That avoids people pushing changes to main which I'm not sure the permissions allow for anyways? Although maybe tags are different

Reply:
Oh, that's a good point. Let me see if I can tag my PR and if that tag persists when it's merged...

Reply:
Yup. The tag can be associated with a commit. The only thing about this is the tag needs to be added to the commit we want users to use. So if changes are made to a branch, the tag needs to be moved.
For this reason, if might be easier to just add the tag after the merge using github. Then we won't have to worry about moving the tag in the case of a PR, or have to remember to add the tag just before merging. I'll update the readme...

Reply:
Nevermind. I don't think one can add tags in github, only releases. The tags are git tags.
I've added some more instructions on removing the tag in github and forcing a new one in the case a tag has to be moved.

Reply:
Are release tags bad though? We have used them in our library repos like product-db-management and they seem to do everything we need (eg. when referencing the repo you can point to a tag rather than branch/commit)

Reply:
Release tags about work, but even for release tags, I think they need to be associated with git tags.